packages/core/src/routes/connector.test.ts

```diff
@@ -257,9 +279,45 @@ describe('connector route', () => {
       );
       expect(deleteConnectorByIds).toHaveBeenCalledWith(['id']);
     });
+
+    it('throws when add more than 1 social connector instance with same target and platform', async () => {
+      loadConnectorFactoriesPlaceHolder.mockResolvedValueOnce([
+        {
+          ...mockConnectorFactory,
+          metadata: {
+            ...mockConnectorFactory.metadata,
+            id: 'id0',
+            platform: ConnectorPlatform.Universal,
+            isStandard: true,
+          },
+        },
+      ]);
+      mockedCountConnectorByConnectorId.mockResolvedValueOnce({ count: 0 });
+      getLogtoConnectorsPlaceHolder.mockResolvedValueOnce([
+        {
+          dbEntry: { ...mockConnector, connectorId: 'id0', metadata: { target: 'target' } },
+          metadata: {
+            ...mockMetadata,
+            id: 'id0',
+            target: 'target',
+            platform: ConnectorPlatform.Universal,
+          },
+          type: ConnectorType.Social,
+          ...mockLogtoConnector,
+        },
+      ]);
+      const response = await connectorRequest.post('/connectors').send({
+        connectorId: 'id0',
+        metadata: { target: 'target' },
+      });
+      expect(response).toHaveProperty('statusCode', 422);
+    });
   });
 
   describe('POST /connectors/:id/test', () => {
+    beforeEach(() => {
```

Comment:
is there any particular reason to have both before and after test cleanups? Would run both reset and clear after the test work?

Reply:
Add this when locally testing and forgot to remove this. Removed now.